Code/Engine/Texture/Image/Formats/TgaFileFormat.cpp

```diff
@@ -156,7 +156,7 @@ ezResult ezTgaFileFormat::WriteImage(ezStreamWriter& stream, const ezImageView&
 
     ezInt32 iRLE = 0;
 
-    ezColorLinearUB pc;
+    ezColorLinearUB pc = {};
```

Comment:
This seems unnecessary.

Reply:
It is not. Because the constructor of ezColorLinearUB  is now = default the compiler can now detect usage of uninitialized ezColorLinearUB instances. "= {}" initializes this instace of ezColorLinearUB to zero and gets rid of the compiler warning.